untitled1/src/main/com/aca/homework/week7/clone/Light.java

```diff
@@ -0,0 +1,25 @@
+package com.aca.homework.week7.clone;
+
+public class Light implements Cloneable {
+    private State state;
+
+    private Light(State state) {
+        this.state = state;
+    }
+
+    @Override
+    protected Light clone() throws CloneNotSupportedException {
+        return (Light) super.clone();
+    }
+
+    public static Light ofLight(State state) {
```

Comment:
please create 2 different methods for each state

Reply:
What is the point of separating these two cases, that is  for  only just to make the interface more understandable legible?